index.md

```diff
@@ -128,7 +117,7 @@ ms.assetid:
                                         <div class="card">
                                             <div class="cardImageOuter">
                                                 <div class="cardImage">
-                                                    <img src="./docs/images/hub/net-gs-3.svg" alt="" />
+                                                    <img src="articles/images/hub/logo_NETframework.svg" alt="" />
```

Comment:
Yes, I can update these. Oddly, the ad cards (at the top) use the articles/images/hub format so I thought there was some build system convention for this docset that needed to be followed. And yes, I can change them to lowercase.